### PR TITLE
Added Hindi and Japanese Translation 

### PR DESCRIPTION
**Pull request by shashankx86**

### CODE REVIEW

**Comment by LarsEJonasson** [comment]
In common.json line 43 and 44 are duplicate translations of "trollboard", and the first lacks the final quotation mark and comma.

In index.json  line 4 lacks the final comma.

In labelling.json line 19 and 65 and lacks the final comma. Line 67 has quoted English terms, but JSON interprets the double quotation marks as a stop. Using escape sequences (a \ before each ") should work.

Line 5 in tos.json should not have a final comma.

**Comment by shashankx86** [comment]
@LarsEJonasson all fixed

**Comment by LarsEJonasson** [comment]
> @LarsEJonasson all fixed and i also can translate to japanese but there isn't any jp folder should i do it?

You still have some formatting issues with tasks.json. If you have Python you can run 'pre-commit run --all-files' on your local computer and it will correct them. For example the spaces between " and : are missing. You also have three initial spaces at each line, but it should be two.

As for more translations, the more the better.

**Comment by shashankx86** [comment]
@LarsEJonasson Done.

**Comment by shashankx86** [comment]
@LarsEJonasson Added jp translation 

image

but here 

image

File: website/public/locales/jp/labelling.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
+  "accurate": "Точное",
```
Comment (AbdBarho) [inline website/public/locales/jp/labelling.json:2]: for the japanese files, if we don't have a translation now, then please use the english one, or remove entirely.

**Comment by shashankx86** [comment]
@LarsEJonasson Done

**Comment by GuilleHoardings** [comment]
Japanese files should be under the `ja` directory, not `jp`.

Thank you for your contribution!

**Comment by shashankx86** [comment]
> Japanese files should be under the ja directory, not jp.

Do i need to fix it? @GuilleHoardings

**Comment by GuilleHoardings** [comment]
I think that it doesn't work if you don't fix it. As far as I know, the framework looks in the directory that matches the locale, and the locale for Japan is `ja`. Have you tested it?

**Comment by shashankx86** [comment]
> I think that it doesn't work if you don't fix it. As far as I know, the framework looks in the directory that matches the locale, and the locale for Japan is `ja`. Have you tested it?

nope

**Comment by shashankx86** [comment]
@GuilleHoardings renamed to ja

**Comment by GuilleHoardings** [comment]
Ok, thank you. I've received another proposal for the Japanase translation, and the submitter merged it with yours. I'll submit as an update when yours is merged.

**Review by andreaskoepf — APPROVED** [review]

**Pull request merged**